test/AssetPool.test.js

```diff
@@ -1190,6 +1191,60 @@ describe("AssetPool", () => {
     })
   })
 
+  describe("cancelNewAssetPoolUpgrade", () => {
+    let newAssetPool
+
+    beforeEach(async () => {
+      const NewUnderwriterToken = await ethers.getContractFactory(
+        "UnderwriterToken"
+      )
+      newUnderwriterToken = await NewUnderwriterToken.deploy(
+        "New Underwriter Token",
+        "newCOV"
+      )
+      await newUnderwriterToken.deployed()
+
+      const NewAssetPoolStub = await ethers.getContractFactory(
+        "NewAssetPoolStub"
+      )
+      newAssetPool = await NewAssetPoolStub.deploy(
+        collateralToken.address,
+        newUnderwriterToken.address
+      )
+      await newAssetPool.deployed()
```

Comment:
Could we move objects related to the new asset pool from `cancelNewPoolUpgrade` and `approveNewAssetPoolUpgrade` to the [main section of AssetPool.test.js](https://github.com/keep-network/coverage-pools/blob/951045379ce38f5e73fd61991f84a946a18b62fc/test/AssetPool.test.js#L25)? They are duplicated in both tests.

Reply:
I'd leave them as-is. It's true they are duplicated but given that they are duplicated only one time, in the context of `approveNewAssetPoolUpgrade` and the rest of the tests in this file do not need them, I think it's better to keep this code as-is.

Reply:
@pdyraga I'm ok with leaving it as is.